src/main/java/org/apache/sysml/api/mlcontext/MLContextUtil.java

```diff
@@ -974,8 +974,15 @@ public static String displayOutputs(Set<String> outputNames, LocalVariableMap sy
 	 * @return the SystemML welcome message
 	 */
 	public static String welcomeMessage() {
-		StringBuilder sb = new StringBuilder();
-		sb.append("\nWelcome to Apache SystemML!\n");
+        StringBuilder sb = new StringBuilder();
+        sb.append("\nWelcome to Apache SystemML!\n Version ");
```

Comment:
Hi @krishnakalyan3. Thank you for working on this!

Would you mind moving `" Version "` to within the `if (info.version() != null) { }` block? Otherwise, if someone is developing using MLContext in an IDE (where no SystemML jar file is available), they see output such as the following in the console:
```
Welcome to Apache SystemML!
 Version 
```
rather than just
```
Welcome to Apache SystemML!
```
Thanks!